tests/testthat/test-as_html.R

```diff
@@ -36,6 +36,11 @@ test_that("code inside Sexpr is evaluated", {
   expect_equal(rd2html("\\Sexpr{1 + 2}"), "3")
 })
 
+test_that("can convert cross links to online documentation url", {
+  expect_equal(rd2html("\\link[base]{library}"),
+               link_remote(label="library", topic="library", package="base"))
```

Comment:
I used the `link_remote` call as the baseline because there is a separate test that `link_remote` gives the desired url. If the url spec changes, that test will be the only one that needs updating.